ivy/functional/ivy/linear_algebra.py

```diff
@@ -628,19 +628,60 @@ def inv(
         floating-point data type determined by :ref:`type-promotion` and must have the
         same shape as ``x``.
 
+    This function conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/
+    latest/extensions/generated/signatures.linalg.inv.html>`
+    in the standard.
+
+    Both the description and the type hints above assumes an array input for simplicity,
+    but this function is *nestable*, and therefore also accepts :code:`ivy.Container`
+    instances in place of any of the arguments.
+
     Examples
     --------
+
```

Comment:
you have an error here:
ivy/functional/ivy/linear_algebra.py:621 in public function `inv`:
D412: No blank lines allowed between a section header and its content ('Examples')